app/constants/global-constants.js

```diff
@@ -89,3 +89,4 @@ export const APP_WINDOW_HEIGHT = 728;
 export const APP_WINDOW_MIN_WIDTH = 1024;
 export const APP_WINDOW_MIN_HEIGHT = 728;
 export const TIMEOUT_BEFORE_APP_PROCESS_EXITS_MS = 12000;
+export const FREEZE_BALANCE_PERCISION = 10e7;
```

Comment:
freeze balance precision same as echo precision, finde already declared echo precision variable and use it